Agents/MOPs_Assembler/README.md

```diff
@@ -0,0 +1,79 @@
+# MOP Assembler: Assembly Model Rescaling, Alignment, and Workflow
+
+## Overview:
+  This project facilitates the rescaling, alignment, and assembly of molecular models using Chemical Building Units (CBUs) for constructing metal-organic polyhedra (MOPs). By leveraging predefined CBUs, the tool processes input data to rescale assembly models, aligns CBUs, and enables MOP construction through JSON-based workflows. The data is organized into JSON files, which describe the atomic and bonding structure of these CBUs.
+
+## Project Structure:
+ - main.py: Entry point of the project for initiating the rescaling, alignment, and MOP construction workflow.
+ - mop_am_gbu_positions.py: Manages the positions of GBUs within the assembly models.
+ - mop_am_rescaler.py: Rescales the assembly model based on CBUs.
+ - mop_cbu_aligner.py: Aligns CBUs to specific positions in the rescaled assembly model.
+ - mop_cbu_find_distance.py: Calculates distances between dummy atoms in CBUs for precise alignment.
+ - mop_jsons2xyz.py: Converts JSON files into XYZ format for further processing.
+ - mop_operations.py: Handles various operations related to the manipulation and adjustment of molecular models.
+
+## Input:
+  CSV File:
```

Comment:
What name does this need to have? Or can you configure this somewhere? In what location should it be? Please add this.